tests/integration_tests/cmd/test_schema.py

```diff
@@ -24,6 +67,65 @@ def test_clean_log(self, class_client: IntegrationInstance):
         assert "apt_update: Default: ``false``. Deprecated in version" in log
         assert "apt_upgrade: Default: ``false``. Deprecated in version" in log
 
+    def test_network_config_schema_validation(
+        self, class_client: IntegrationInstance
+    ):
+        content_responses = {
+            NET_CFG_V1: {"out": "Valid schema /root/net.yaml"},
+            NET_CFG_V1_INVALID: {
+                "out": "Invalid network-config /root/net.yaml",
+                "err": (
+                    "network: Additional properties are not allowed"
+                    " ('junk' was unexpected)"
+                ),
+                "annotate": NET_V1_ANNOTATED,
+            },
+        }
+        if CURRENT_RELEASE.series == "noble":
```

Comment:
```suggestion
        if CURRENT_RELEASE >= NOBLE:
```
with import from tests.integration_tests.releases

Reply:
Thanks grabbed that builtin comparison thoughout.